02-IDE_exercises/solutions/03a-project/min_max_number.py

```diff
@@ -4,26 +4,26 @@
 #              integers from user
 
 print("How many numbers would you like to enter?")
-num_int = int(input())
+num_int: int = int(input())
 # The first while loop ensures that the user enters at least one number
 # if, the user enters anything less than 1, it will prompt the user to enter something greater than one
 
 while num_int < 1:
     print("Please enter at least one number")
-    num_int = int(input())
+    num_int: int = int(input())
 else:
     print("Please enter", num_int, "numbers.")
 
-    first_int = int(input())
+    first_int: int = int(input())
     #  the first number is the largest as well as the smallest number
-    min = first_int
-    max = first_int
+    min: int = first_int
+    max: int = first_int
 
     # if the number of ints requested by user is greater than one, we'll initialize
     # a while loop
 
     while num_int > 1:
-        n = int(input())
+        n = int(input())  # type: int
```

Comment:
can you note type hint `n`?

I haven't checked with mypy but at the very least I'm expecting you can type hint like:
```python
n: int
n = int(input())
```
is the syntax to type hint `n` like `n:int = int(input())` illegal?